tests/toolkit/infile_converters_tests.cc

```diff
@@ -2,6 +2,8 @@
 
 #include "cyclus.h"
 
+#ifdef CYCLUS_WITH_PYTHON
```

Comment:
Do all these tests require python? Including the XML<->JSON?

Reply:
No, the XML<->JSON does not require python.  I moved the `#ifdef CYCLUS_WITH_PYTHON` to encompass only the tests requiring python.  Thanks for catching that @gonuke!